.azure/scripts/setup-kind.sh

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/env bash
+set -xe
+
+rm -rf ~/.kube
+
+# There is a bug in 0.24.0 - https://github.com/kubernetes-sigs/kind/issues/3713
+KIND_VERSION=${KIND_VERSION:-"v0.23.0"}
```

Comment:
https://github.com/kubernetes-sigs/kind/issues/3713 looks like it should be resolved in 0.25

Reply:
I can try with the 0.25

Reply:
The only issue is that the oldest K8s version in 0.25.0 is 1.26 -> so we can test on newer version than the "oldest supported by Strimzi operators" (1.25), but I thought that we want to keep it consistent in all repos, so I kept it as it is.
But if it's not a problem and we want to test on the latest K8s, then I can update the version of Kind as well :)

Reply:
Ahh, ok. We do not need to test with 1.25 here I think. 1.26 would be fine. But it would matter more in the operators repo if we want to use the same version there as well.